getting-started/templates/systemlink-values.yaml

```diff
@@ -591,22 +591,22 @@ dashboardhost:
 
 ## Grafana provisioning
 ##
-grafanaprovisioning:
-  grafanaDashboardsProvisioning:
+# grafanaprovisioning:
```

Comment:
Do we expect customers to need to modify this section? If so, consider adding an ATTENTION comment. If not, consider whether we can just delete it to simplify the template.